adapter/oci/app_client.go

```diff
@@ -0,0 +1,74 @@
+package oci
+
+import (
+	"context"
+	"fmt"
+	"github.com/fnproject/cli/adapter"
+	"github.com/oracle/oci-go-sdk/functions"
+	"github.com/spf13/viper"
+	"os"
+)
+
+type AppClient struct {
+	client *functions.FunctionsManagementClient
+}
+
+func (a AppClient) CreateApp(app *adapter.App) (*adapter.App, error) {
+	//TODO: call OCI client
```

Comment:
Hmmm, this is a bit hard to review without any context. These bits are unimplemented. Is this part of a bigger or multistage strategy for making these changes? If these are really meant to be unimplemented and return null for the short term, then maybe we can some comments explaining the purpose and strategy.

Reply:
This is a WIP. The initial round of code review was to get feedback on class design and adapter code structures. As we move to fully implement the "unimplemented bits" things will get in better shape before the final code review.